README.md

```diff
@@ -6,37 +6,113 @@
 
 Special educational needs and disability (demand and costs) model in development to be used on MastodonC's [Witan](http://www.mastodonc.com/products/witan/) city decision-making platform or as a standalone modelling library.
 
+## Running the Model
 
-## Development tools
+At the top level of the send repo runs
```

Comment:
Simply the top level directory

Reply:
I think it might just be a typo? Do you mean "At the top level of the send repo run"? 

Even when winding back to cider 0.16.0 this didn't method didn't work for me